packages/lib-classifier/src/components/Classifier/hooks/usePanoptesUserSession.js

```diff
@@ -0,0 +1,19 @@
+import {
+  usePanoptesUser,
+  useProjectPreferences,
+  useProjectRoles
+} from '@hooks'
+
+export default function usePanoptesUserSession({ authClient, projectID }) {
+  const { user, loading: userLoading } = usePanoptesUser(authClient)
```

Comment:
Non-blocking: Because `userProjectPreferences()` and `useProjectRoles()` return an `isLoading` variable, would it make sense for `usePanoptesUserSession()` and `usePanoptesUser()` to also return `isLoading` rather than "loading" for consistency? Or perhaps the other way around changing the former two to `loading`.

Reply:
I've updated `usePanoptesUser` to use a similar API to [the `useSWR` API.](https://swr.vercel.app/docs/api)